### DISCUSSION TITLE
Aufero Laser1 will only "print" "laser" in a straight line - one axis

### DISCUSSION THREAD

**Discussion by WickedFunX** [arkypita/LaserGRBL#1778] [Q&A]
I've watched videos, read everything, made svg files....  When I hit create, play - it burns my design in a straight line and not the text "picture".  Uggghhhh Frustrated!

**Comment by WickedFunX** [comment]
![5x5 web](https://user-images.githubusercontent.com/100539822/155914997-2e74414b-2c2c-48cd-b45d-a819e545df16.jpg)
![wood blank](https://user-images.githubusercontent.com/100539822/155915010-1932af42-a688-462d-b5ec-9bfcd503273a.jpg)
The WickedFun logo is what it was supposed to engrave on the wood blank.  And all it did was engrave - actually cut through the wood blank, a straight line.....

**Comment by arkypita** [comment]
Sounds more an hardware/wiring issue instead of a LaserGrbl issue. Does the jog arrow works or not?

**Comment by WickedFunX** [comment]
If you mean where there is a picture of a house and arrows all around it.  Yes I can move the laser head around with those arrows.

So like maybe I need to buy a new machine??

**Comment by WickedFunX** [comment]
I wrote this back on the page:

If you mean where there is a picture of a house and arrows all around it. Yes I can move the laser head around with those arrows.

So like maybe I need to buy a new machine??


I hope you will help me - I've tried so many things....


...LISA...
________________________________

**Comment by WickedFunX** [comment]
I got it fixed.

**Comment by Tribo-2dot0** [comment]
How?
